File: Service/ScanCare/RMA.Service.ScanCare.Contracts/Entities/DocumentType.cs
using System;
using System.Collections.Generic;

namespace RMA.Service.ScanCare.Contracts.Entities
{
    public class DocumentType
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string ValidDays { get; set; }
        public string CreatedBy { get; set; }
        public DateTime CreatedDate { get; set; }
        public string ModifiedBy { get; set; }
        public DateTime ModifiedDate { get; set; }
        public string Manager { get; set; } // Manager (length: 255)

        public List<Document> Documents { get; set; }
        public List<DocumentSetDocumentType> DocumentSetDocumentTypes { get; set; }

    }
}

File: Web/RMA-Web-Shell/projects/shared-models-lib/src/lib/common/document-type.ts
import { DocumentSetDocumentType } from './document-set-document-type';

export class DocumentType {
    id: number;
    name: string;
    validDays: string;
    createdBy: string;
    createdDate: Date | string;
    modifiedBy: string;
    modifiedDate: Date | string;
    manager: string; // Manager (length: 255)

    documents: Document[];
    documentSetDocumentTypes: DocumentSetDocumentType[];
}
